libeos-updater-util/types.c

```diff
@@ -36,6 +36,9 @@ static const GDBusErrorEntry eos_updater_error_entries[] = {
   { EOS_UPDATER_ERROR_WRONG_CONFIGURATION, "com.endlessm.Updater.Error.WrongConfiguration" },
   { EOS_UPDATER_ERROR_NOT_OSTREE_SYSTEM, "com.endlessm.Updater.Error.NotOstreeSystem" },
   { EOS_UPDATER_ERROR_FETCHING, "com.endlessm.Updater.Error.Fetching" },
+  { EOS_UPDATER_ERROR_MALFORMED_AUTOINSTALL_SPEC, "com.endlessm.Updater.Error.MalformedAutoinstallSpec" },
```

Comment:
These error entries should only be added later on, in the commit which introduces the first use of them.

Reply:
Yep, that's a good point. Moved.